recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -39,11 +40,16 @@ class TensorflowLiteConan(ConanFile):
 
     short_paths = True
 
+    @property
+    def _min_cppstd(self):
+        return "17"
+
     @property
     def _compilers_minimum_version(self):
         return {
             "gcc": "8",
             "Visual Studio": "15.8",
+            "msvc": "191",
```

Comment:
why MSVC version rather than toolset is here?
See https://cmake.org/cmake/help/latest/variable/MSVC_TOOLSET_VERSION.html

Reply:
Why do you think that compiler.version is related to this link?

Here the content of settings.yml:

```yml
    msvc:
        version: [170, 180, 190, 191, 192, 193]
        update: [null, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
        runtime: [static, dynamic]
        runtime_type: [Debug, Release]
        cppstd: [null, 14, 17, 20, 23]
        toolset: [null, v110_xp, v120_xp, v140_xp, v141_xp]
```

compiler.version=191 when compiler=msvc means Visual Studio 2017 for conan.